addons/base_vat/models/res_partner.py

```diff
@@ -356,3 +360,22 @@ def check_vat_tr(self, vat):
             return int(vat[9]) == c1 and int(vat[10]) == c2
 
         return False
+
+    def _fix_vat_number(self, vat):
+        vat_country, vat_number = self._split_vat(vat)
+        if vat_country == 'be':
```

Comment:
I know it wasn't in the specs but since stdnum has a compact function available for a lot of countries it would be a pitty to include only the belgian formatting of VAT, don't you think?

```
...
if hasattr(stdnum.get(vat_country), 'compac'):
    return vat_country.upper() + getattr(stdnum.get(vat_country), 'compac')(vat_number)
return vat
```
or something like that

Reply:
Actually not as easy as that:
https://github.com/arthurdejong/python-stdnum/tree/master/stdnum
For belgium: from stdnum.be.vat import compact
For france: from stdnum.fr.tva import compact
For mexican: from stdnum.mx.rfc import compact
and so on for each country.

I'm not sure on how to do that?